src/cascadia/WindowsTerminal/WindowEmperor.cpp

```diff
@@ -5,14 +5,10 @@
 #include "WindowEmperor.h"
 
 #include "../inc/WindowingBehavior.h"
-
 #include "../../types/inc/utils.hpp"
-
 #include "../WinRTUtils/inc/WtExeUtils.h"
-
 #include "resource.h"
 #include "NotificationIcon.h"
-#include <til/env.h>

```

Comment:
There didn't seem to be a system to the newlines between the includes, so I removed them. Let me know if I should revert this.